src/terminal/parser/ut_parser/OutputEngineTest.cpp

```diff
@@ -2011,4 +2018,63 @@ class StateMachineExternalTest final
 
         pDispatch->ClearState();
     }
+
+    TEST_METHOD(TestSetClipboard)
+    {
+        auto dispatch = std::make_unique<StatefulDispatch>();
+        auto pDispatch = dispatch.get();
+        auto engine = std::make_unique<OutputStateMachineEngine>(std::move(dispatch));
+        StateMachine mach(std::move(engine));
+
+        mach.ProcessString(L"\x1b]52;;Zm9v\x07");
+        VERIFY_ARE_EQUAL(L"foo", pDispatch->_copyContent);
+
+        pDispatch->ClearState();
+
+        mach.ProcessString(L"\x1b]52;;Zm9vDQpiYXI=\x07");
```

Comment:
The bot doesn't like this line.

You should be able to add something like this to https://github.com/microsoft/terminal/blob/master/.github/actions/spell-check/patterns/patterns.txt: 
`\bmach\.ProcessString\(L"\x1b]52;;Zm9vDQpiYXI=\x07"\)`

(A future version of the bot should actually identify this line...)

Reply:
Fixed. Thanks!